lib/Net/DRI/DRD/GoDaddy/MZB.pm

```diff
@@ -0,0 +1,291 @@
+## Domain Registry Interface, GoDaddy::MZB Driver
+##
+## Copyright (c) 2021 Paulo Jorge <paullojorgge@gmail.com>. All rights reserved.
+##
+## This file is part of Net::DRI
+##
+## Net::DRI is free software; you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation; either version 2 of the License, or
+## (at your option) any later version.
+##
+## See the LICENSE file that comes with this distribution for more details.
+####################################################################################################
+
+package Net::DRI::DRD::GoDaddy::MZB;
+
+use strict;
+use warnings;
+
+use base qw/Net::DRI::DRD/;
+
+####################################################################################################
+=pod
+
+=head1 NAME
+
+Net::DRI::DRD::GoDaddy::MZB - GoDaddy::MZB Driver for Net::DRI
+
+=head1 DESCRIPTION
+
+Additional MZB extension GoDaddy::MZB
+GoDaddy::MZB use a dedicated server and the following standard extensions. Please see the test files for more examples.
+
+=head2 Standard extensions:
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::secDNS> urn:ietf:params:xml:ns:secDNS-1.1
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::GracePeriod> urn:ietf:params:xml:ns:rgp-1.0
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::LaunchPhase> urn:ietf:params:xml:ns:launch-1.0
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::IDN> urn:ietf:params:xml:ns:idn-1.0
+
+=head3 L<Net::DRI::Protocol::EPP::Extensions::AllocationToken> urn:ietf:params:xml:ns:allocationToken-1.0
+
+=head2 Custom extensions (NeuLevel)
```

Comment:
@paullojorgge Does MZB have all of these other extensions enabled?

Reply:
Yes, it runs in the same server. POD updated - my bad!

Reply:
Thanks Paulo, but all of the POD below this is still incorrect, is it not?